facebook/facebook-json.c

```diff
@@ -314,3 +314,34 @@ gboolean fb_json_str_chk(const json_value *json, const gchar *name,
     *val = jv->u.string.ptr;
     return TRUE;
 }
+
+/**
+ * Backslash-escapes a string to make it safe for json. The returned string
+ * should be freed with #g_free() when no longer needed.
+ *
+ * *param input The string to escape
+ *
+ * @return The resulting string, or NULL on error.
+ **/
+gchar *fb_json_escape_string(const gchar *input)
```

Comment:
Call this `fb_json_str_escape()` for consistency with the other functions in `facebook-json` and `facebook-util`.